controls_rewrite_folder/driver_interface_error_handling.h

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+#include <cmath>
+#pragma once
+//everson shen and aleeza ali zar
+namespace ctrl{
+
+template <typename T>
+bool isInRange(T fractionIn){
+
+    return (fractionIn >= 0 && fractionIn <= 1);
+}
+
+
+template <typename T>
+bool b_DriverInterfaceError(T DI_V_AccelPedalPos1, T DI_V_AccelPedalPos2, T DI_FrontBrakePressure, T DI_RearBrakePressure, T DI_V_RawSteeringAngle){
+    const T TOLERANCE = 0.00000000000001; // Define a small tolerance for floating-point comparisons
```

Comment:
We do not need this tolerance since it does not matter if the pedals are exactly 0.1 apart.